style.md

```diff
@@ -339,7 +339,7 @@ A single blank line appears:
 
  2. Between statements, as _needed_ to organize the code into logical subsections.
 
- 3. _Optionally_ before the first statement in a function, or before the first member of after the last member of a class (neither encourage nor discouraged).
+ 3. _Optionally_ before the first statement in a function, or both before the first member and after the last member of a class (neither encourage nor discouraged).
```

Comment:
This wording is confusing. The presence of a space before the first member does not require that one be present after the last.

Reply:
Sure, applied a fix.